src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Constants/ActivityConstants.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+
+namespace Microsoft.Extensions.Configuration.AzureAppConfiguration
+{
+    internal class ActivityConstants
```

Comment:
```suggestion
    internal static class ActivityNames
```

Constants is to vague to be useful for anything. I would also recommend not having a folder named constants.

Reply:
![image](https://github.com/user-attachments/assets/9a7eeae1-aeea-45e8-bd03-284e093df78d)

![image](https://github.com/user-attachments/assets/198f45bf-75c9-48f8-9ecd-8f5320f1a0c5)

I followed the existing pattern. Should I also update them?

Reply:
No, my suggestion is just go with a new class ActivityNames that is clear what it's for.